spec/features/user_edits_a_review_spec.rb

```diff
@@ -10,13 +10,13 @@
     click_on "Edit Review"
 
     fill_in "Title", with: "Edited Title"
-    fill_in "Body", with: "Edited Body"
+    fill_in "Body", with: "#{'y' * 50}"
     select 2, from: "Rating"
     attach_file("Photo", "spec/fixtures/images/pizza_slice.jpeg")
     click_on "Update Review"
 
     expect(page).to have_content "Edited Title"
-    expect(page).to have_content "Edited Body"
+    expect(page).to have_content "#{'y' * 50}"
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.